contracts/transmuter/libraries/LibOracle.sol

```diff
@@ -126,7 +134,15 @@ library LibOracle {
                 );
             }
             return quotePrice;
-        } else return _targetPrice;
+        } else if (readType == OracleReadType.STABLE) return BASE_18;
+        else if (readType == OracleReadType.NO_ORACLE) return baseValue;
+        else if (readType == OracleReadType.WSTETH) return STETH.getPooledEthByShares(1 ether);
```

Comment:
I am not sure I understand why we merge target and read functions? Like all the ETH functions will be useless

Reply:
yes they'll be useless for targets, but goal is to only keep one type overall -> so we don't have to do the distinction between normal oracles and targets